### PR TITLE
DMAC / DMACMAN additions

### PR DESCRIPTION
**Pull request by kernle32dll**
Some collected DMAC stuff I was working on.

- DMAC D3, D5 and D8 TADR writing and reading 
- DMAC DPCR3
- DMACMAN calls 14 - 19

### CODE REVIEW

File: Source/iop/Iop_Dmac.h [hunk 1]
```diff
@@ -67,6 +67,7 @@ namespace Iop
 		{
 			DPCR = 0x1F8010F0,
 			DPCR2 = 0x1F801570,
+			DPCR3 = 0xBF8015F0,
```
Comment (jpd002) [inline Source/iop/Iop_Dmac.h:70]: Should be a physical address (ie.: `0x1F8015F0`)

Comment (kernle32dll) [reply Source/iop/Iop_Dmac.h:70]: Oh boy, my bad

**Comment by jpd002** [comment]
It's missing state load/save for `m_D3_TADR` and `m_D5_TADR` but otherwise, seems good!

**Comment by jpd002** [comment]
Thanks!

**Pull request merged**